src/main/java/io/confluent/kafkarest/AdminClientWrapper.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.kafkarest;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.ConfigEntry;
+import org.apache.kafka.clients.admin.DescribeClusterResult;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.common.Node;
+import org.apache.kafka.common.TopicPartitionInfo;
+import org.apache.kafka.common.config.ConfigResource;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Properties;
+import java.util.TreeSet;
+import java.util.Vector;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import io.confluent.kafkarest.entities.Partition;
+import io.confluent.kafkarest.entities.PartitionReplica;
+import io.confluent.kafkarest.entities.Topic;
+import io.confluent.rest.exceptions.RestServerErrorException;
+import jersey.repackaged.com.google.common.collect.ImmutableList;
+
+
+public class AdminClientWrapper {
+
+  private AdminClient adminClient;
+  private int initTimeOut;
+
+  public AdminClientWrapper(KafkaRestConfig kafkaRestConfig) {
+    Properties properties = new Properties(kafkaRestConfig.getAdminProperties());
+    properties.put(KafkaRestConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaRestConfig.bootstrapBrokers());
+    adminClient = AdminClient.create(properties);
+    this.initTimeOut = kafkaRestConfig.getInt(KafkaRestConfig.KAFKACLIENT_INIT_TIMEOUT_CONFIG);
+  }
+
+  public List<Integer> getBrokerIds() {
+    List<Integer> brokerIds = new Vector<>();
+    DescribeClusterResult clusterResults = adminClient.describeCluster();
+    try {
+      Collection<Node>
```

Comment:
general comment: i often find your formatting inconsistent and unusual. Here, I'd put the `nodeCollection =` variable name on line 60 (consistent with what you do on line 87.